funnel/templates/notifications/layout_email.html.jinja2

```diff
@@ -1,45 +1,511 @@
+<head>
+    <meta charset="utf-8"> {# utf-8 works for most cases #}
+    <meta name="viewport" content="width=device-width"> {# Forcing initial-scale shouldn't be necessary #}
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"> {# Use the latest (edge) version of IE rendering engine #}
+    <meta name="x-apple-disable-message-reformatting">  {# Disable auto-scale in iOS 10 Mail entirely #}
+    <meta name="format-detection" content="telephone=no,address=no,email=no,date=no,url=no"> {# Tell iOS not to automatically link certain text strings. #}
+    <meta name="color-scheme" content="light">
+    <meta name="supported-color-schemes" content="light">
+    <title></title> {#   The title tag shows in email notifications, like Android 4.4. #}
+
+    {# What it does: Makes background images in 72ppi Outlook render at correct size. #}
+    <!--[if gte mso 9]>
+    <xml>
+        <o:OfficeDocumentSettings>
+            <o:AllowPNG/>
+            <o:PixelsPerInch>96</o:PixelsPerInch>
+        </o:OfficeDocumentSettings>
+    </xml>
+    <![endif]-->
+
+    {# Outlook / @font-face : BEGIN #}
+
+    {# Desktop Outlook chokes on web font references and defaults to Times New Roman, so we force a safe fallback font. #}
+    <!--[if mso]>
+        <style>
+            * {
+                font-family: sans-serif !important;
+            }
+        </style>
+    <![endif]-->
+
+    {# Outlook / @font-face : END #}
+
     {% block stylesheet -%}
-      <style type="text/css">
-        .btn, a.btn:link, a.btn:visited {
-          background-color: #428bca;
-          border-radius: 3px;
-          border: 1px solid #357ebd;
-          color: white;
-          margin-bottom: 6px;
-          padding: 6px 12px;
-          text-decoration: none;
-          white-space: nowrap;
-        }
-        .btn-bar {
-          margin: 2em 0;
-        }
-        blockquote {
-          margin-left: 0;
-          padding-left: 1em;
-          border-left: 3px solid #ccc;
-        }
-      </style>
+    {# CSS Reset : BEGIN #}
+    <style>
+
+        /* What it does: Tells the email client that only light styles are provided but the client can transform them to dark. A duplicate of meta color-scheme meta tag above. */
+        :root {
+          color-scheme: light;
+          supported-color-schemes: light;
+        }
+
+        /* What it does: Remove spaces around the email design added by some email clients. */
+        /* Beware: It can remove the padding / margin and add a background color to the compose a reply window. */
+        html,
+        body {
+            margin: 0 auto !important;
+            padding: 0 !important;
+            height: 100% !important;
+            width: 100% !important;
+        }
+
+        /* What it does: Stops email clients resizing small text. */
+        * {
+            -ms-text-size-adjust: 100%;
+            -webkit-text-size-adjust: 100%;
+        }
+
+        /* What it does: Centers email on Android 4.4 */
```

Comment:
Are these Android 4 styles still relevant? Most users will be on minimum Android 9. Really old devices will be Android 8.

Reply:
Removed.